authors/apps/articles/tests/test_likes.py

```diff
@@ -0,0 +1,131 @@
+        article_id = response.data['id']
+        url = '/api/articles/{}/like/'.format(article_id)
+        response = self.client.post(
+            url, HTTP_AUTHORIZATION=self.auth_header, format="json")
+        self.assertEqual(response.data['success'],"You have successfully liked this article.")
+
+    
+
+    def test_user_like_article_twice_status(self):
+        """
+        Method tests liking article twice status code
+        """
+        url = reverse('articles-list-create')
+        response = self.client.post(
+            url, self.create_article_data, HTTP_AUTHORIZATION=self.auth_header, format="json")
+        article_id = response.data['id']
+        url = '/api/articles/{}/like/'.format(article_id)
+        response = self.client.post(
+            url, HTTP_AUTHORIZATION=self.auth_header, format="json")
+        response = self.client.post(
+            url, HTTP_AUTHORIZATION=self.auth_header, format="json")
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+
+    def test_user_like_article_twice_error_message(self):
+        """
+        Method tests liking article twice error message
+        """
+        url = reverse('articles-list-create')
+        response = self.client.post(
+            url, self.create_article_data, HTTP_AUTHORIZATION=self.auth_header, format="json")
+        article_id = response.data['id']
+        url = '/api/articles/{}/like/'.format(article_id)
+        response = self.client.post(
+            url, HTTP_AUTHORIZATION=self.auth_header, format="json")
+        response = self.client.post(
+            url, HTTP_AUTHORIZATION=self.auth_header, format="json")
+        self.assertEqual(response.data['message'], "You already liked this article")
+
+    def test_user_dislike_article_status(self):
+        """
+        Method tests the status on dislike article response
+        """
+        url = reverse('articles-list-create')
+        response = self.client.post(
+            url, self.create_article_data, HTTP_AUTHORIZATION=self.auth_header, format="json")
+        article_id = response.data['id']
+        url = '/api/articles/{}/dislike/'.format(article_id)
+        response = self.client.post(
+            url, HTTP_AUTHORIZATION=self.auth_header, format="json")
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    def test_user_dislike_article_message(self):
+        """
+        Method tests disliking the article response message
+        """
+        url = reverse('articles-list-create')
+        response = self.client.post(
+            url, self.create_article_data, HTTP_AUTHORIZATION=self.auth_header, format="json")
+        article_id = response.data['id']
+        url = '/api/articles/{}/dislike/'.format(article_id)
+        response = self.client.post(
+            url, HTTP_AUTHORIZATION=self.auth_header, format="json")
+        self.assertEqual(response.data['success'],"You have successfully disliked this article.")
+
+    
```

Comment:
Please remove these blank lines

Reply:
Thanks for the reminder @sanya-kenneth. I have now linted the code.